src/util/colors.d.ts

```diff
@@ -0,0 +1,45 @@
+declare module 'vuetify/es5/util/colors' {
+  interface Color {
+    base: string;
+    lighten5: string;
+    lighten4: string;
+    lighten3: string;
+    lighten2: string;
+    lighten1: string;
+    darken1: string;
+    darken2: string;
+    darken3: string;
+    darken4: string;
+    accent1?: string;
```

Comment:
I'd have something like 
```js
interface BaseColor {
  // base, lighten, darken
}

interface Color extends BaseColor {
  // accent
}
```

So that it's clear which ones don't have accents.

Reply:
Or maybe generics if you want to get fancy with it (idk if this is a good idea but):

```js
interface Color<A = string> {
  // base, lighten, darken as strings
  accent1: A
  ...
}
...
export const deepOrange: Color
export const brown: Color<undefined>
```